### PR TITLE
Links to data used for DataTable storybook examples

### PR DESCRIPTION
**Pull request by IanKBovard**
#### What does this PR do?
Adds this link to relevant DataTable storybook examples.
https://github.com/grommet/grommet/blob/master/src/js/components/DataTable/stories/data.js

#### Where should the reviewer start?
All DataTable storybook examples except InfiniteScrollDataTable.js

#### What testing has been done on this PR?
Checked links in storybook examples.

#### How should this be manually tested?
Not needed

#### Any background context you want to provide?
Addresses this comment on slack.
https://grommet.slack.com/archives/C04LMHN59/p1581918046158100

#### Do the grommet docs need to be updated?
No 
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: src/js/components/DataTable/stories/ControlledGrouped.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ const ControlledGroupedDataTable = () => {
         }}
         sortable
       />
+      <Box margin={{ top: 'medium' }}>
+        <Anchor
```
Comment (ShimiSun) [inline src/js/components/DataTable/stories/ControlledGrouped.js]: You can move the `margin` directly on Anchor, and then remove the Box usage.

File: src/js/components/DataTable/stories/Simple.js [hunk 2]
```diff
@@ -10,6 +10,12 @@ const SimpleDataTable = () => (
   <Grommet theme={grommet}>
     <Box align="center" pad="large">
       <DataTable columns={columns} data={DATA} step={10} />
+      <Box margin={{ top: 'medium' }} align="center">
+        <Anchor
```
Comment (ShimiSun) [inline src/js/components/DataTable/stories/Simple.js]: Can you check if we can move the `margin={{ top: 'medium' }} align="center"` to the Anchor level? and then eliminate the Box? Anchor has `alignSelf`.

Comment (ShimiSun) [reply src/js/components/DataTable/stories/Simple.js]: Or maybe we don't really need the `align`, to keep it simple.

Comment (IanKBovard) [reply src/js/components/DataTable/stories/Simple.js]: changes made. I added align only because I think it looks nicer.

**Comment by ShimiSun** [comment]
Looking at this on storybook, I feel like the link should be on the code so only if the user is looking for the code he will be alert by the link.
Screen Shot 2020-03-05 at 7 50 04 AM

I was thinking maybe we should just add a comment line on lines 6 or 7 with a link to the data, and make the example UI look cleaner. what do you think?

**Comment by IanKBovard** [comment]
changes made

File: src/js/components/DataTable/stories/Clickable.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ import { Grommet, Box, DataTable } from 'grommet';
 import { grommet } from 'grommet/themes';
 
 import { columns, DATA } from './data';
+// Source code for the data can be found here
+// https://github.com/grommet/grommet/blob/master/src/js/components/DataTable/stories/data.js
```
Comment (ShimiSun) [inline src/js/components/DataTable/stories/Clickable.js]: Move to line 6 before the Data is imported.

**Pull request merged**